### ISSUE TITLE
Diagram plugin can not load

### ISSUE THREAD

**Issue by Britaliope** [jdbranham/grafana-diagram#57]
When i create a new dashboard i can't add a diagram in it : when i click on the button the following error appear : 

```
Plugin component error Error: (SystemJS) illegal character
	
	Evaluating https://graph.resel.fr/public/plugins/jdbranham-diagram-panel/libs/mermaid/dist/mermaidAPI.js?bust=1500739119869
	Error loading https://graph.resel.fr/public/plugins/jdbranham-diagram-panel/libs/mermaid/dist/mermaidAPI.js as "./libs/mermaid/dist/mermaidAPI" from https://graph.resel.fr/public/plugins/jdbranham-diagram-panel/diagramControl.js
Trace de la pile :
(SystemJS) illegal character
	
	Evaluating https://graph.resel.fr/public/plugins/jdbranham-diagram-panel/libs/mermaid/dist/mermaidAPI.js?bust=1500739119869
	Error loading https://graph.resel.fr/public/plugins/jdbranham-diagram-panel/libs/mermaid/dist/mermaidAPI.js as "./libs/mermaid/dist/mermaidAPI" from https://graph.resel.fr/public/plugins/jdbranham-diagram-panel/diagramControl.js
```

**Comment by jdbranham** [comment]
Hi @Britaliope -
Did you figure out why it was failing?

**Comment by Keleir** [comment]
When I installed this plugin via `grafana-cli plugins install jdbranham-diagram-panel`, 
then restart grafana-server, It doesnot shown on pluings page.

Error occurs in the log `/var/log/grafana/grafana.log` : 
```
t=2017-11-22T18:56:03+0800 lvl=eror msg="Plugins: Failed to load plugin json file: 
/var/lib/grafana/plugins/jdbranham-diagram-panel/dist/plugin.json,  err: json: 
cannot unmarshal string into Go struct field PluginInfo.author of type 
plugins.PluginInfoLink"
```
But when I install it with a [zip file](https://grafana.com/api/plugins/jdbranham-diagram-panel/versions/1.3.0/download) and move it into the plugins directory, It works well.

```
t=2017-11-22T18:56:03+0800 lvl=info msg="Registering plugin" logger=plugins 
name=Diagram
```

My grafana version: **4.6.2**

**Comment by ben-turner** [comment]
There's a problem with the plugin.json file. I worked around it by changing the author line in /var/lib/grafana/plugins/jdbranham-diagram-panel/dist/plugin.json.

Before: 
```
{
...
  "info": {
  ...
    "author": "Jeremy Braham .... ",
  ...
...
}
```
After:
```
{
...
  "info": {
  ...
    "author": {
      "name": "Jeremy Branham",
      "email": "jeremy@savantly.net",
      "url": "https://www.linkedin.com/in/jeremybranham/"
    },
  ...
...
}
```

**Comment by jdbranham** [comment]
new version released which addresses this issue v1.4.4
